playbooks/roles/elasticsearch/defaults/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+elasticsearch_apt_repo_url:
+"http://packages.elasticsearch.org/elasticsearch/1.5/debian"
```

Comment:
This needs to be on the same line as the var, so:

elasticsearch_apt_repo_url: "http://packages.elasticsearch.org/elasticsearch/1.5/debian"